robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowApplication.java

```diff
@@ -213,6 +217,15 @@ public Resources getResources() {
   }
 
   /**
+   * Creates a {@link RoboAttributeSet} for the given {@link Attribute}(s)
+   */
+  public RoboAttributeSet createAttributeSet(Attribute... attrs) {
+    List<Attribute> attributesList = Lists.newArrayList(attrs);
+    return new RoboAttributeSet(attributesList,
+        shadowOf(RuntimeEnvironment.application.getAssets()).getResourceLoader());
```

Comment:
I think this should probably move to ShadowAssetManager at some point but this is OK for now. I say this because maybe different Contexts (which have different AssetManagers) may have different assets.